icons/xiaomi.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
```

Comment:
```suggestion
<svg role="image" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
```

Reply:
My bad, thought I'd done that.

Reply:
No problem, in fact my suggestion was [not correct either](https://travis-ci.com/simple-icons/simple-icons/jobs/242298724#L233) 😅 I fixed that for you in 3601733